solutions/USACO feb20-triangles.mdx

```diff
@@ -0,0 +1,62 @@
+---
+id: usaco-1011
+source: USACO 2020 Bronze February 2020
+title: Triangles
+author: Mrinall Umasudhan
+---
+
+[Official Analysis (C++)](http://www.usaco.org/index.php?page=viewproblem2&cpid=1011)
+
+### Implementation
+
+Brute force all possible right triangles by looping over all triples of points and checking whether they form a right triangle. If so, compute the area and find the maximum over all right triangles.
+
```

Comment:
maybe make a note that the brute force takes N^3

Reply:
I just copied and pasted the editorial from the original solution page on the guide but I will add that as well if you would like.

Reply:
Done